### PR TITLE
Added the workflow and files for validation-dashboard

### PR DESCRIPTION
**Pull request by BALAGA-GAYATRI**
This PR contains:
1. Workflow for triggering validation action for multiple repos.
2. Necessary files and scripts.
Screenshot 2021-08-03 at 6 15 17 PM

### CODE REVIEW

File: .github/workflows/repo-validation-checks.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      id: create-json
+      uses: jsdaniell/create-json@1.1.2
+      with:
+        name: "validation-dashboard/result.json"
+        json: "${{steps.checks_validation.outputs.validationResult}}"
+    - name: validation result and  Commit file
+      id: commit1
+      continue-on-error: true
+      run: |
+           git config --local user.email "balaga-gayatri@github.com"
```
Comment (kanika1894) [inline .github/workflows/repo-validation-checks.yml:27]: let's use the standard config credentials that we use for other workflows?

Comment (BALAGA-GAYATRI) [reply .github/workflows/repo-validation-checks.yml]: Done !

File: .github/workflows/repo-validation-checks.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      with:
+        GITHUB_TOKEN: ${{ secrets.PAT_TOKEN }}
+        repositories: 'login,webapps-deploy,aci-deploy,build-vm-image,functions-action,functions-container-action,mysql,postgresql,powershell,sql-action,appservice-settings,variable-substitution,policy-compliance-scan,k8s-create-secret,k8s-deploy,k8s-lint,manage-azure-policy,setup-kubectl,aks-set-context,k8s-bake,static-web-apps-deploy,get-keyvault-secrets,AppConfiguration-Sync,k8s-set-context,setup-helm,container-scan,docker-login,publish-security-assessments,arm-deploy,cli'
+    - name: create-json
+      id: create-json
+      uses: jsdaniell/create-json@1.1.2
+      with:
+        name: "validation-dashboard/result.json"
+        json: "${{steps.checks_validation.outputs.validationResult}}"
+    - name: validation result and  Commit file
```
Comment (kanika1894) [inline .github/workflows/repo-validation-checks.yml:23]: Let's. mention JSON here as we mentioned HTML in the other job?

File: .github/workflows/repo-validation-checks.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+name: "repo-validation"
+
+on: workflow_dispatch
```
Comment (kanika1894) [inline .github/workflows/repo-validation-checks.yml:3]: Add cron also for twice a week

File: validation-dashboard/json2htmlrunner.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+table_attributes = {"style": "width:100%"}
+table = json2html.convert(json = infoFromJson)
+f = open( 'docs/validation-result.html', 'w' )
+f.write( table )
+f.close()
+f = open('docs/validation-result.html', 'a')  
+f.write("""<link rel="stylesheet" href="validation-result.css" type="text/css" media="all">
+           <script>
+            window.onload = function color() {
+                console.log('test')
```
Comment (kanika1894) [inline validation-dashboard/json2htmlrunner.py:21]: We can remove this console statement

File: .github/workflows/repo-validation-checks.yml (hunk 2 above)
Comment (BALAGA-GAYATRI) [reply .github/workflows/repo-validation-checks.yml]: done

File: .github/workflows/repo-validation-checks.yml (hunk 3 above)
Comment (BALAGA-GAYATRI) [reply .github/workflows/repo-validation-checks.yml]: done

File: validation-dashboard/json2htmlrunner.py (hunk 4 above)
Comment (BALAGA-GAYATRI) [reply validation-dashboard/json2htmlrunner.py]: removed

**Pull request merged**